packages/edit-widgets/src/blocks/widget-area/editor.scss

```diff
@@ -21,6 +21,10 @@
 	.block-list-appender.wp-block {
 		width: initial;
 	}
+	// Override theme custom widths for blocks.
+	.editor-styles-wrapper .wp-block.wp-block.wp-block.wp-block.wp-block {
```

Comment:
Should we just do `!important` here, or would that be too much specificity?

Reply:
Yeah, I prefer to go with the lowest specificity option where possible - this should be enough, and it it proves not to be we can always change it to `!important` later.